src/components/Modal/ModalSelectAssets.tsx

```diff
@@ -85,8 +86,8 @@ const ModalSelectAssets = () => {
       }
       getAssetList.push({ ...value, isNotSelectable })
     })
-    setAssetList(getAssetList)
-    setAssetListWithBalances(getAssetListWithBalances)
+    setAssetList(tieNativeToWrapToken(getAssetList))
```

Comment:
Maybe it would be more reliable to do it globally and in one place if possible. (If there is one global storage)

Reply:
Well, yes and no, the problem here is this from tokens perspective this is different entity which we consume from our solver and for them they are different entities as well and if we bundle them at our some root entry point, mutate them to singleton then we could have some trouble in all our outputs, make revers procedure.